commands/fork.go

```diff
@@ -65,7 +68,10 @@ func fork(cmd *Command, args *Args) {
 
 	originRemote, err := localRepo.OriginRemote()
 	if err != nil {
-		utils.Check(fmt.Errorf("Error creating fork: %s", err))
+		originRemote, err = localRepo.RemoteByName("upstream")
```

Comment:
On the above line, you can use `localRepo.MainRemote()` instead of `OriginRemote` if you want to select the first remote that matches: `upstream`, `github`, or `origin`, [in that order](https://github.com/github/hub/blob/e43d204db5233b51169fc755ab979bb9c4ebe663/github/remote.go#L13). Then you don't need to add this separate `RemoteByName()` call.